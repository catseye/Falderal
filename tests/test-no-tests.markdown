Falderal Test for No Tests Found
--------------------------------

Since these blocks are not indented correctly, `falderal` should not
load tests from them.  (Ideally it should produce warnings though.)

  -> Functionality "Cat" is implemented by
  -> shell command "python3 cat.py"

Cat cats.

| meow
= meow

        | purr
        | prrr
        | prreow
        = purr
        = prrr
        = prreow
