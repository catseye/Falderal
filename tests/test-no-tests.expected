FalderalLintingError: No functionalities were found in any of the test documents
